sandbox/docker-compose.yml

```diff
@@ -42,6 +49,11 @@ services:
     environment:
       - CUE_FRAME_LOG_DIR=$CUE_FRAME_LOG_DIR
     command: --datasource.cue-data-source.jdbc-url=jdbc:postgresql://db/cuebot --datasource.cue-data-source.username=cuebot --datasource.cue-data-source.password=$POSTGRES_PASSWORD
+    logging:
```

Comment:
Should this logging configuation be removed form this `docker-compose.yml` as it's now in `docker-compose.monitoring.yml`?